.script/tests/KqlvalidationsTests/CustomTables/SenservaPro_CL.json

```diff
@@ -60,6 +60,10 @@
     {
       "Name": "Group_s",
```

Comment:
Custom table schema having column names like column_x. Custom table should have column with final field names.